x-pack/plugins/apm/server/index.test.ts

```diff
@@ -7,27 +7,23 @@
 
 /* eslint-disable @typescript-eslint/naming-convention */
 
-import { APMOSSConfig } from 'src/plugins/apm_oss/server';
 import { APMXPackConfig } from '.';
 import { mergeConfigs } from './index';
 
 describe('mergeConfigs', () => {
   it('merges the configs', () => {
-    const apmOssConfig = {
+    const apmConfig = {
       transactionIndices: 'apm-*-transaction-*',
       spanIndices: 'apm-*-span-*',
       errorIndices: 'apm-*-error-*',
       metricsIndices: 'apm-*-metric-*',
-    } as APMOSSConfig;
-
-    const apmConfig = {
       ui: { enabled: false },
       enabled: true,
       metricsInterval: 2000,
       agent: { migrations: { enabled: true } },
     } as APMXPackConfig;
 
-    expect(mergeConfigs(apmOssConfig, apmConfig)).toEqual({
+    expect(mergeConfigs(apmConfig)).toEqual({
       'apm_oss.errorIndices': 'logs-apm*,apm-*-error-*',
```

Comment:
Should it still return `apm_oss`?

Reply:
I'll continue to update tests and mocks to remove all references of apm_oss from tests.